src/components/Dashboard/RecentPastScheduledObs.vue

```diff
@@ -0,0 +1,68 @@
+<script setup>
+import { useRouter } from 'vue-router'
+import { useObsPortalDataStore } from '../../stores/obsPortalData.js'
+import { useConfigurationStore } from '../../stores/configuration.js'
+import { onMounted, ref } from 'vue'
+import { fetchApiCall } from '../../utils/api.js'
+import ObservationList from './ObservationList.vue'
+
+const router = useRouter()
+const obsPortalDataStore = useObsPortalDataStore()
+const completedObservations = obsPortalDataStore.completedObservations
+const configurationStore = useConfigurationStore()
+const recentRTI = ref([])
+const recentCompleted = ref([])
+
+// Filter out scheduled observations that are older than two weeks
+const twoWeeksAgo = new Date() - 14 * 24 * 60 * 60 * 1000
+const recentObservations = Object.values(completedObservations).filter(observation => new Date(observation.end) > twoWeeksAgo)
+const recentCompletedObservations = recentObservations.filter(observation => observation.state === 'COMPLETED')
```

Comment:
If you think a lot of these filters might be used in the future and are useful they could be getters in the obsPortalStore, if that store keeps track of observations. Makes the store the go to authority for components asking about observations

Reply:
I'm trying to think of where else I may need this and nothing comes to mind. I'll keep a mental note to make it a getter if I access these values again : )